PowerPointLabs/PowerPointLabs/Models/PowerPointMagnifyingSlide.cs

```diff
@@ -146,9 +145,33 @@ private void AddZoomSlideCroppedPicture(float magnifyRatio = 1.0f)
             cropShape.Select();
             PowerPoint.Selection sel = Globals.ThisAddIn.Application.ActiveWindow.Selection;
             PowerPoint.Shape croppedShape = CropToShape.Crop(zoomSlideCopy, sel, magnifyRatio: magnifyRatio);
-            croppedShape.Cut();
+            try
+            {
+                string tempFilePath = FileDir.GetTemporaryPngFilePath();
+                Utils.GraphicsUtil.ExportShape(croppedShape, tempFilePath);
+                zoomSlideCroppedShapes = _slide.Shapes.AddPicture2(tempFilePath,
+                    Office.MsoTriState.msoFalse,
+                    Office.MsoTriState.msoTrue,
+                    0,
+                    0);
+                croppedShape.Delete();
+                try
+                {
+                    FileDir.DeleteFile(tempFilePath);
+                }
+                catch (Exception)
+                {
+                    // If the file cannot be deleted, we continue without deletion.

```

Comment:
Is there some information logging that should be done? What is the expected outcome of this block of code?

Reply:
This block of code is needed so that an an exception from this particular exception will not be caught by the catch block below. If the file cannot be deleted due to some permissions issues, we will just leave the file in the user's directory.

Reply:
We do have a logger for pptlabs, you could use it.